Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/OTPAuthorizationResult.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.datadonation
+
+import androidx.annotation.Keep
+import com.google.gson.annotations.SerializedName
+import org.joda.time.Instant
+import java.util.UUID
+
+@Keep
+data class OTPAuthorizationResult(
+    @SerializedName("uuid")
+    val uuid: UUID,
+    @SerializedName("expirationDate")
+    val expirationDate: Instant,
```

Comment:
This needs to be `Instant?` because unsuccessful authentications won't have an `expireDate`, or?

Reply:
According to the change from Max, we might don't even need the `expirationDate`?

Reply:
removed the field